std_assert_test.go

```diff
@@ -0,0 +1,249 @@
+package objx_test
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"testing"
+)
+
+type nonfatal struct{}
+type fatal struct{}
+
+var assert nonfatal
+var require fatal
+
+func (nonfatal) fail(t *testing.T, msg string, msgAndArgs ...interface{}) bool {
+	if len(msgAndArgs) > 0 {
+		msg = fmt.Sprintf(msg, msgAndArgs...)
+	}
+	t.Helper()
+	t.Errorf(msg)
+	return false
+}
+
+func (fatal) fail(t *testing.T, msg string, msgAndArgs ...interface{}) bool {
+	if len(msgAndArgs) > 0 {
+		msg = fmt.Sprintf(msg, msgAndArgs...)
+	}
+	t.Helper()
+	t.Fatalf(msg)
+	return false
+}
+
+func (a nonfatal) Equal(t *testing.T, expected, actual interface{}, msgAndArgs ...interface{}) bool {
+	t.Helper()
+	if !reflect.DeepEqual(expected, actual) {
+		return a.fail(t, "not equal:\nexpected: %#v\nactual:   %#v", expected, actual)
+	}
+	return true
+}
+func (a fatal) Equal(t *testing.T, expected, actual interface{}, msgAndArgs ...interface{}) bool {
```

Comment:
There is an issue with all fatal methods.

Here I'm talking about a compatibility issue with testify that is pointless as you remove testify dependency.

But I have to mention it.

testify/require asserters don't return bool

So here (and everywhere) it should be this


```suggestion
func (a fatal) Equal(t *testing.T, expected, actual interface{}, msgAndArgs ...interface{}) {
```